css/properties/align-content.json

```diff
@@ -237,7 +237,9 @@
               },
               "firefox": [
```

Comment:
Why is there no change of align-content on Firefox for Android when they are for justify-content and place-content.

I understand that the bug only applies to Style and will land on Fx for Android only when Style will land there (which Firefox already?), but I don't understand how align-content may be different from the others?

Reply:
D'oh, this was an oversight. Fixing.